app/workers/lighthouse/document_upload.rb

```diff
@@ -4,6 +4,7 @@
 require 'timeout'
 require 'lighthouse/benefits_documents/worker_service'
 
+# [wipn8923] replaces app/workers/evss/document_upload.rb ?
```

Comment:
This is the main entry point into the mobile team's code I used in https://github.com/department-of-veterans-affairs/vets-api/pull/13835/commits/2fb7784c80147a0aaa84d647c4f6fbcbeadfa1ee